src/main/java/com/ibm/eventstreams/connect/mqsource/JMSReader.java

```diff
@@ -351,7 +352,8 @@ private boolean connectInternal() {
                 Thread.sleep(reconnectDelayMillis);
             }
             catch (InterruptedException ie) {
-                ;
+                log.error("Thread.sleep failed in connectInternal()", jmse);
```

Comment:
This PR should not include other stuff too. Please restrict it to configuration of a CSP option only.

Reply:
returned ";"  :)